caliban_toolbox/build.py

```diff
@@ -43,14 +45,15 @@ def compute_cell_size(npz_file, method='median', by_image=True):
             the cell size across the entire npz is returned
 
     Returns:
-        average_sizes: list of typical cell size in NPZ
+        average_sizes: list of typical cell size in NPZ. If no cells, returns None
 
     Raises: ValueError if invalid method supplied
     Raises: ValueError if data does have len(shape) of 4
     """
 
-    valid_methods = set(['median', 'mean'])
-    if method.lower() not in valid_methods:
+    valid_methods = {'median', 'mean'}
+    method = method.lower()
```

Comment:
You can cast as a string to make sure that `.lower()` doesn't throw an attribute error.

```suggestion
    method = str(method).lower()
```